edge-modules/ModuleRestarter/src/Program.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace ModuleRestarter
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    class Program
+    {
+        static readonly ILogger Log = Logger.Factory.CreateLogger<Program>();
+        static readonly string ServiceClientConnectionString = Settings.Current.ServiceClientConnectionString;
+        static readonly string DeviceId = Settings.Current.DeviceId;
+        static readonly string DesiredModulesToRestartCSV = Settings.Current.DesiredModulesToRestartCSV;
+        static readonly int RandomRestartIntervalInMins = Settings.Current.RandomRestartIntervalInMins;

```

Comment:
It is not random for restart interval; it is random for which module to be restarted.  Therefore I think this parameter should be named as "Restart interval In Mins".  However, do we want to restart with random intervals?

Reply:
I think that restarting on fixed intervals is fine. I agree that a name change would help clarity here.